docs/src/api/class-page.md

```diff
@@ -3953,6 +3953,36 @@ Receives the [ConsoleMessage] object and resolves to truthy value when the waiti
 ### param: Page.waitForConsoleMessage.callback = %%-java-wait-for-event-callback-%%
 * since: v1.9
 
+## async method: Page.waitForDialog
+* since: v1.34
+* langs: java, python, csharp
+  - alias-python: expect_dialog_message
+  - alias-csharp: RunAndWaitForDialog
```

Comment:
I think we don't want RunAndWaitForDialogAsync in .NET, only waitForDialogAsync is enough.